boost/interprocess/detail/managed_open_or_create_impl.hpp

```diff
@@ -310,7 +310,7 @@ class managed_open_or_create_impl
                //File existing when trying to create, but non-existing when
                //trying to open, and tried MaxCreateOrOpenTries times. Something fishy
                //is happening here and we can't solve it
-               throw interprocess_exception(error_info(corrupted_error));
+              throw interprocess_exception(error_info(corrupted_error), "file exists when create but not when open");
```

Comment:
```suggestion
               throw interprocess_exception(error_info(corrupted_error), "do_create_else_open: file exists when trying to create, but not when trying to open");
```

Reply:
For consistency with the other cases where you mention the function name explicitly.

Reply:
done